content/javascript/concepts/strings/strings.md

```diff
@@ -21,13 +22,12 @@ In JavaScript, multiple strings can be concatenated or joined together using the
 
 In the example below, multiple strings and variables containing string values have been concatenated. After the code block is run, the `displayText` variable will contain the concatenated string:
 
-```js
+```codebyte/javascript
 let service = 'credit card';
 let month = 'May 30th';
 let displayText = 'Your ' + service + ' bill is due on ' + month + '.';
 
 console.log(displayText);
-// Output: Your credit card bill is due on May 30th.
 ```
```

Comment:
You don't need to replace the existing code with a new one. You have to add a `##Codebyte Example` block, which is supposed to be at the end of the entry and should cover all the three topics above which are: Concatenation, interpolation, and escaping characters

Reply:
Okay. I have made some edits based on your feedback. Take a look. Strings.md is attached to this comment.

Reply:
If you don't see my updated strings.md let me know.